### PR TITLE
Added Card Graph for temp history information

### PR DESCRIPTION
**Pull request by neilmacintyre**
Added graph that is working well. However there appears to be a memory leak somewhere that is occasionally created when opening the Temp card. Still have to find it though

### CODE REVIEW

**Comment by neilmacintyre** [comment]
one other thing I have all noticed is that the the math for the time/x axis is off. Will also fix this for the main branch.

File: app/build.gradle [hunk 1]
```diff
@@ -19,10 +21,16 @@ android {
...
+    maven { url "https://jitpack.io" }
+}
 
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
     compile 'com.android.support:appcompat-v7:20.0.0'
     compile 'com.github.gabrielemariotti.cards:cardslib-core:2.0.1'
+    //compile 'com.github.nmac143:MPAndroidChart:a053d894c8'
+    //compile 'com.github.PhilJay:MPAndroidChart:v2.1.3'
+    compile 'com.github.nmac143:MPAndroidChart:ab4bb825fc'
```
Comment (neilmacintyre) [inline app/build.gradle:34]: added the chart library

File: app/src/main/java/com/mariogrip/octodroid/HistoryObject.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
+package com.mariogrip.octodroid;
```
Comment (neilmacintyre) [inline app/src/main/java/com/mariogrip/octodroid/HistoryObject.java:1]: holds the Json objects in a Java instance

File: app/src/main/java/com/mariogrip/octodroid/UpdateGraph.java [hunk 3]
```diff
@@ -0,0 +1,199 @@
+package com.mariogrip.octodroid;
```
Comment (neilmacintyre) [inline app/src/main/java/com/mariogrip/octodroid/UpdateGraph.java:1]: This class is used to update and create the graph

File: app/src/main/java/com/mariogrip/octodroid/UpdateGraph.java [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+                }
+                break;
+
+        }
+
+
+        return  tempHistory;
+    }
+
+    private static ArrayList<String> getXVals(int length) {
```
Comment (neilmacintyre) [inline app/src/main/java/com/mariogrip/octodroid/UpdateGraph.java:174]: This method creates the Xaxis values. The math to the test string value works for about 10 minutes and then appears to break (needs to be fixed in a future commit)

**Comment by mariogrip** [comment]
Awesome!

**Pull request merged**